lib/puppet/type/openldap_database.rb

```diff
@@ -101,4 +101,18 @@ def should_to_s( newvalue )
     newvalues(:true, :false)
     defaultto(:true)
   end
+
+  newproperty(:mirrormode) do
```

Comment:
I'm having problems with this. When I define it with ```:boolean => true``` and I try to modify a resource putting this property to false (to a resource with this property already set to true), nothing is done: puppet does not recognize that the change is needed.